Add licenses

matthiasbeyer opened the pull request:
###### Motivation for this change

Adds licenses for:

pkgs/applications/networking/browsers/arora/default.nix
pkgs/development/compilers/aspectj/default.nix
pkgs/development/interpreters/clisp/default.nix
pkgs/games/atanks/default.nix
pkgs/tools/package-management/checkinstall/default.nix

because I was bored.

###### Things done

- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

related to #43716

matthiasbeyer committed:
arora: Add license

Signed-off-by: Matthias Beyer <mail@beyermatthias.de>

matthiasbeyer committed:
aspectj: Add license

Source: https://git.eclipse.org/c/aspectj/org.aspectj.git/tree/docs/build.xml#n7

Signed-off-by: Matthias Beyer <mail@beyermatthias.de>

matthiasbeyer committed:
atanks: Add license

Source: https://sourceforge.net/p/atanks/atanks/ci/master/tree/COPYING

Signed-off-by: Matthias Beyer <mail@beyermatthias.de>

matthiasbeyer committed:
checkinstall: Add license

Source: http://checkinstall.izto.org/docs/LICENSE

Signed-off-by: Matthias Beyer <mail@beyermatthias.de>

Mic92 commented on pkgs/development/interpreters/clisp/default.nix:
```diff
@@ -99,5 +99,6 @@ stdenv.mkDerivation rec {
     platforms = stdenv.lib.platforms.unix;
     # problems on Darwin: https://github.com/NixOS/nixpkgs/issues/20062
     broken = stdenv.hostPlatform.isDarwin || stdenv.hostPlatform.isAarch64;
+    licenses = stdenv.lib.licenses.gpl2;
```
```suggestion
    license = stdenv.lib.licenses.gpl2;
```

matthiasbeyer committed:
clisp: Add license

Source: http://ftp.gnu.org/pub/gnu/clisp/release/latest/clisp-2.49.tar.gz
File: GNU-GPL

Signed-off-by: Matthias Beyer <mail@beyermatthias.de>
